src/styles/base/_tables.scss

```diff
@@ -16,8 +16,13 @@ td {
 	}
 }
 
-tr:last-child td,
-tr:last-child th {
+tbody:last-child > tr:last-child td {
```

Comment:
I think these styles are _super_ specific, and assume that all table are marked up with a `tbody`/`tfoot` etc. Is there a border collapse issue?